### ISSUE TITLE
Tooltip isn't WCAG compliant

### ISSUE THREAD

**Issue by StudioVDS** [highcharts/highcharts#23553]
#### Expected behaviour

- I should be able to select the text in the tooltip (1.3.2 I think)

#### Actual behaviour

I can't select the content of the tool tip.

#### Actions
I tried overwriting some setting with css, like the pointer-event en user-select. And setting the tooltip to useTML and stickOnContact but none of it worked?

**Comment by magdalena-gut** [comment]
Hi @StudioVDS 
You were on the right track with useHTML, stickOnContact and CSS overrides - to make it fully work, you also need to wrap the tooltip content in a custom formatter and add `onmousedown="event.stopPropagation()` so you can drag to select text.
```
Highcharts.AST.allowedAttributes.push('onmousedown');

Highcharts.chart('container', {
  tooltip: {
    useHTML: true, 
    stickOnContact: true,
    formatter: function () {
      return `<div style="user-select:text" onmousedown="event.stopPropagation()">
        SAMPLE TEXT ${this.point.y}
      </div>`;
    }
  },
  series: [{ data: [1, 2, 3, 4, 5, 6] }]
});

```
Demo: https://jsfiddle.net/BlackLabel/0e95my6h/

---
As this is not a bug, I will go ahead and close this ticket.
If I missed something, please don’t hesitate to leave a comment.

If you have more questions, feel free to contact our support team through one of the channels listed here:
https://www.highcharts.com/support/

**Comment by StudioVDS** [comment]
Hi Magdalena,

Thanks for your input looks promissing. But I couldn't get this to work because I'm not allowed to use inline javascript because of my CSP settings. I tried getting the event outside Highchart with no luck till now. Also if I'm right the `event` part is deprecated.

I fixed it for now by adding this before the highcharts code, is this a good solution?:

```
  //make tooltip selectable
  document.addEventListener("mousedown", function(e) {
    if (e.target.closest(".selectable-tooltip")) {
      e.stopPropagation();
    }
  }, true);
```

**Comment by magdalena-gut** [comment]
Hi, yes this is a good alternative, thank you for sharing!

Just to keep it here for others, here’s a full improved version:
Demo: https://jsfiddle.net/BlackLabel/pymfva96/
